app/javascript/react/components/TrailTile.js

```diff
@@ -1,23 +1,35 @@
-import React from "react";
-import { Link } from "react-router-dom";
+import React, { useState } from "react";
+import { Redirect } from "react-router-dom";
 
 const TrailsTile = (props) => {
+    const { trail }  = props
+
+    const [redirect, setRedirect] = useState(false)
+
+    const tileClick = () => {
+        setRedirect(true)
+    }
+
     return (
-        <div className="cell small-6 card">
-            <div className="card-divider">
-                <h2>
-                    <Link to={`/trails/${props.id}`}> {props.name} </Link>
-                </h2>
-            </div>
-            <div className="card-section">
-                <h4>
-                    {`${props.city}`}
-                </h4>
-                <p>
-                    {`${props.zip}`}
-                </p>
+            <div className="cell small-6 card" onClick={tileClick}>
+                {redirect && <Redirect to={`/trails/${trail.id}`}/>}
+                <div className="card-divider">
+                    <h2>
+                        {trail.name}
+                    </h2>
+                </div>
+                <div className="card-section">
+                    <div className="trail-image">
+                        <img src={trail.image_url} alt={`Image of ${trail.name}.`}/>
+                    </div>
+                    <h4>
+                        {`${trail.city}`}
+                    </h4>
+                    <p>
+                        {`${trail.zip}`}
```

Comment:
this zip and the city don't need to be string interpolated, we just just render the values using JSX `{ }`